pennylane/capture/__init__.py

```diff
@@ -135,10 +135,45 @@ def _(*args, **kwargs):
 )
 from .capture_qnode import qnode_call
 
+# by defining this here, we avoid
+# E0611: No name 'AbstractOperator' in module 'pennylane.capture' (no-name-in-module)
+# on use of from capture import AbstractOperator
+AbstractOperator: type
+AbstractMeasurement: type
+qnode_prim: "jax.core.Primitive"
+
 
 def __getattr__(key):
     if key == "AbstractOperator":
         from .primitives import _get_abstract_operator  # pylint: disable=import-outside-toplevel
 
         return _get_abstract_operator()
+
+    if key == "AbstractMeasurement":
+        from .primitives import _get_abstract_measurement  # pylint: disable=import-outside-toplevel
+
+        return _get_abstract_measurement()
+
+    if key == "qnode_prim":
+        from .capture_qnode import _get_qnode_prim  # pylint: disable=import-outside-toplevel
+
+        return _get_qnode_prim()
+
     raise AttributeError(f"module 'pennylane.capture' has no attribute '{key}'")
+
+
+__all__ = (
+    "disable",
+    "enable",
+    "enabled",
+    "CaptureMeta",
+    "ABCCaptureMeta",
+    "create_operator_primitive",
+    "create_measurement_obs_primitive",
+    "create_measurement_wires_primitive",
+    "create_measurement_mcm_primitive",
+    "qnode_call",
+    "AbstractOperator",
+    "AbstractMeasurement",
+    "qnode_prim",
+)
```

Comment:
So this is needed in addition to the 
```
AbstractOperator: type
AbstractMeasurement: type
qnode_prim: "jax.core.Primitive"
```
above?

Reply:
This was what tried first, so I'm not 100% sure if both were needed, but this is also just a good idea to add. I was needing to make sure the linters knew that `AbtractOperator` etc was actually provided by this module.

Reply:
Alright, sounds good! I was just thinking that maintaining this is an additional requirement for future dev, but we can still modify if it annoys us in the future :)

Reply:
We should actually start defining `__all__` for each module and submodule, as it provides a clear reference to "this is what the module provides".  Everything else is "use at your own risk".

Defining `__all__` would make sure each module has a clear, well defined API.

Reply:
I see, that's fair! :)